core/src/main/kotlin/org/evomaster/core/EMConfig.kt

```diff
@@ -1151,6 +1154,11 @@ class EMConfig {
             " on the JVM.")
     var instrumentMR_EXT_0 = true
 
+    @Cfg("Execute instrumentation for method replace with category MONGO." +
+            " Note: this applies only for languages in which instrumentation is applied at runtime, like Java/Kotlin" +
+            " on the JVM.")
+    var instrumentMR_MONGO = true
```

Comment:
it should be marked as `@Experimental`, with `false` as default value. only once stable we can put it on by default